packages/zowe-explorer/src/uss/actions.ts

```diff
@@ -223,9 +223,16 @@ export async function uploadFile(node: IZoweUSSTreeNode, doc: vscode.TextDocumen
 
         // if new api method exists, use it
         if (ZoweExplorerApiRegister.getUssApi(prof).putContent) {
-            await ZoweExplorerApiRegister.getUssApi(prof).putContent(doc.fileName, ussName, {
+            const task: ITaskWithStatus = {
+                percentComplete: 0,
+                statusMessage: localize("uploadFile.putContents", "Uploading USS file"),
+                stageName: 0, // TaskStage.IN_PROGRESS - https://github.com/kulshekhar/ts-jest/issues/281
+            };
+            const options: zowe.IUploadOptions = {
                 encoding: prof.profile.encoding,
```

Comment:
this looks like an existing issue but `encoding` doesn't exist on zowe.IUploadOptions, I think it should be changed to `localEncoding`
![image](https://user-images.githubusercontent.com/49491949/101408555-92989100-38aa-11eb-85b1-b1ca02a16e85.png)

Reply:
then the `prof.profile.encoding` may need to be `prof.profile.encoding.toString()` since zosmf encoding value is a number